index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="bg-white">
+        <div class="logo">
+          <a href="#">Logo</a>
+        </div>
+<!-- <nav>
+          <div class="menu-icon">
+            <img src="images/Union.png" alt="menu button" />
+          </div>
+        </nav> -->
```

Comment:
- [ ] as the previous reviewer requested, kindly add the nav button to match the Figma design

![image](https://user-images.githubusercontent.com/89282221/186408223-3f6e7464-e254-4ddb-b2c5-1d0cfb3dbc3f.png)

Reply:
Done.